infogami/infobase/_json.py

```diff
@@ -22,7 +22,7 @@ def unicodify(d):
         return {k: unicodify(v) for k, v in iteritems(d)}
     elif isinstance(d, list):
         return [unicodify(x) for x in d]
-    elif isinstance(d, str):
+    elif isinstance(d, bytes):
```

Comment:
do we need a separate check for `str` still for python2?

Reply:
No, because legacy Python makes no distinction between bytes and str while Python 3 strictly enforces that distinction.
%  `python2 -c "print(bytes == str)" ` # True
%  `python3 -c "print(bytes == str)" ` # False